frontend/packages/ceph-storage-plugin/src/components/create-storage-system/external-storage/ibm-flashsystem/type.ts

```diff
@@ -37,4 +37,5 @@ export type FlashSystemState = {
   password: string;
   endpoint: string;
   poolname: string;
+  volmode?: string;
```

Comment:
```suggestion
  volmode: string;
```
state should not have optional types